server/src/services/store/templates/draft-2-command-line-tool.hbs

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "{{ id }}",
+    "class": "CommandLineTool",
+    "label": "{{ label }}",
+    "description": "{{ description }}",
```

Comment:
Isn't the id some unique RDF identifier?
Regarding description, there's a description input field in the file creation form. If it's supposed to be lengthy, we can either remove it from the form, or make it a larger text area. Depends on the actual importance of it being filled out.